### PR TITLE
PEOPLE-101 unpend and fix specs

### PR DESCRIPTION
**Pull request by mczuchnowski**
**Jira:**
https://netguru.atlassian.net/browse/PEOPLE-101

**Changes:**
- unpend scheduling and user specs
- remove specs that change non-existing features
- refactor other specs to work with current React-based app

Part 3 of our big undertaking to bring back feature specs to People app

### CODE REVIEW

File: spec/features/users_spec.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@
     visit '/users'
   end
 
-  xit "shows user role" do
+  it "shows user role" do
```
Comment (netguru-hound) [inline spec/features/users_spec.rb]: Prefer single-quoted strings when you don't need string interpolation or special symbols.

File: spec/features/user_profile_spec.rb [hunk 2]
```diff
@@ -1,9 +1,9 @@
 require 'spec_helper'
 
 describe 'profile', js: true do
-  let!(:junior) { create(:role, name: 'junior', priority: 2) }
-  let!(:developer) { create(:role, name: 'developer', priority: 1) }
-  let(:position) { create(:position, role: junior) }
+  let!(:junior_role) { create(:role, name: 'junior') }
```
Comment (mateuszgorniak) [inline spec/features/user_profile_spec.rb]: `let` would be enough. What about traits?

Comment (mczuchnowski) [inline spec/features/user_profile_spec.rb]: True, I think we have a trait for that now. These specs are soooo old...

Comment (mateuszgorniak) [inline spec/features/user_profile_spec.rb]: I see only https://github.com/netguru/people/blob/master/spec/factories/role.rb#L22 but I'm not sure you want to use exactly the same value (with `technical: true`)

**Pull request merged**